invenio_files_rest/alembic/e172c837b036_add_indexes.py

```diff
@@ -0,0 +1,64 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2023 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Add indexes."""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "e172c837b036"
+down_revision = "a29271fd78f8"
```

Comment:
Another alembic recipe was merged after creating this one, so we should update this down_revision to point to that one to keep the correct flow.